dev-starts3.sh

```diff
@@ -200,3 +200,20 @@ do
            --s3port $s3port --fault_injection true $fake_params --loading_indicators --getoid true
   ((++counter))
 done
+
+#restart s3background services
+is_producer_running=1
+$USE_SUDO systemctl is-active s3backgroundproducer 2>&1 > /dev/null || is_producer_running=0
```

Comment:
@abhilekh268 , wrt to your question "if we intend to stop background services at this point ,can we simply stop them without this conditional check."

we can do that.But checking to see if the service is active and then stopping it would be proper way to do it